### PR TITLE
Header fix menu

### PR DESCRIPTION
**Pull request by benthorner**

### CODE REVIEW

**Comment by andrewgarner** [comment]
The styling when you hover over the service name isn't quite right:

screen shot 2018-04-13 at 10 45 20

**Comment by benthorner** [comment]
Fixed - can you take another look?

File: app/assets/stylesheets/govuk-overrides.scss [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
-
-body {
-  font-family: $helvetica-regular;
-}
+#global-header {
+  padding: 5px 0;
 
-// Homepage header overides__________________
+  .header-global {
+    @media (min-width: 769px) {
```
Comment (andrewgarner) [inline app/assets/stylesheets/govuk-overrides.scss]: How about we use the `$desktop-breakpoint` variable from https://github.com/alphagov/govuk_frontend_toolkit/blob/2011e77436fccd31f32a0a975a2bf4cc3b961495/stylesheets/_conditionals.scss#L28 ?

Comment (benthorner) [reply app/assets/stylesheets/govuk-overrides.scss]: That's better - that must be where they got that magic number from originally!

**Pull request merged**